core/src/test/java/org/fao/geonet/kernel/security/openidconnect/OIDCRoleProcessorTest.java

```diff
@@ -242,6 +242,17 @@ public Map<String, Object> createComplexClaims() {
         return claims;
     }
 
+    public Map<String, Object> createSingleValuedRoleClaims() {
```

Comment:
method could be private and live at the end of the file ?

Reply:
:+1:

Reply:
I'm wondering if I should do so for all the other methods of the test class which are currently public but seem to be only called once in the test class.